### ISSUE TITLE
Any advice, please.

### ISSUE THREAD

**Issue by TaeniKim** [cchallu/nbeatsx#8]
Hi. I read your paper very well.
I want to do two experiments.

1. Multivariate regression
  You said "The NBEATSx model offers a solution to the multivariate regression problem" in your paper.
  I will try with a multivariate datasets.

2. A probabilistic forecast, not a point  forecasting
  I'm going to apply 'Quntile regression'.
  I wonder if it can be implemented without difficulty in the current architecture.
  For example, simple 'loss function, ..' modifications, etc.

Please let me know if you have any advice.
Thank you.

**Comment by kdgutier** [comment]
Hi @TaeniKim,

1. Multivariate regression problem:
We are tackling the problem P(y_[t+1:t+H] | y_[t-L:t], X_[t+1:t+H], S), where H is the forecast horizon and L are the lags, y is the target variable and X are temporal exogenous variables available at the prediction time.

2. We have the [Multi-Quantile Loss](https://github.com/Nixtla/neuralforecast/blob/main/neuralforecast/losses/pytorch.py#L306-L365)  available in the neuralforecast library. 

You can use it to model marginal distributions with a non parametric approach.
In the past we adapted the outputs of the MLPs in the N-BEATS architecture to track the number of quantiles that you want to predict.
